### PR TITLE
Make throttling params qps and burst configurable

### PR DESCRIPTION
**Pull request by benjaminkz**
*Issue #, if available:*

*Description of changes:* Kubernetes Go client uses a default value of 5 and 10 for QPS and burst. As a result, our customers may experience client-side rate limiting. This pull request makes them configurable to our customers.

*Related issues:* https://github.com/aws/secrets-store-csi-driver-provider-aws/issues/136

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -218,6 +218,25 @@ envsubst < deployment/private-installer.yaml | kubectl apply -f -
 
 The AWS Secrets Manager and Config Provider provides compatibility for legacy applications that access secrets as mounted files in the pod. Security conscious applications should use the native AWS APIs to fetch secrets and optionally cache them in memory rather than storing them in the file system.
 
+### Client-Side Rate-Limitting to Kubernetes API server
+
+To mount each secret on each pod, the AWS CSI provider lookups the region of the pod and the role ARN associated with the service account by calling the K8s APIs. You can increase the value of qps and burst if you notice the provider is throttled by client-side limit to the API server. Add your custom qps and burst to the definition of `csi-secrets-store-provider-aws` in `aws-provider-installer.yaml`
+
+```ymal
```
Comment (dreamer-89) [inline README.md:225]: Typo ?
```suggestion
```yaml
```

File: main.go [hunk 2]
```diff
@@ -58,6 +60,9 @@ func main() {
 		klog.Fatalf("Can not get cluster config. error: %v", err)
 	}
 
+	cfg.QPS = float32(*qps)
```
Comment (dreamer-89) [inline main.go:63]: Thanks @benjaminkz for the fix. Would it be possible to add a test that adds `qps` and `burst` as configurable parameters ?

**Review by dreamer-89 — COMMENTED** [review]
Thanks @benjaminkz for the fix. Would it be possible to add a test for this change

File: README.md (hunk 1 above)
Comment (benjaminkz) [reply README.md:225]: Will fix that in new commit.

File: main.go (hunk 2 above)
Comment (benjaminkz) [reply main.go:63]: I printed the value of them to the console, where I can see in eks event logging. They were well received by the provider.

**Pull request closed without merging**